osu.Game/Overlays/Rankings/Tables/CountriesTable.cs

```diff
@@ -63,5 +60,20 @@ public CountriesTable(int page, IReadOnlyList<CountryStatistics> rankings)
                 Text = $@"{item.Performance / Math.Max(item.ActiveUsers, 1):N0}",
             }
         };
+
+        private class CountryName : OsuSpriteText
+        {
+            public CountryName(Country country)
+            {
+                Font = OsuFont.GetFont(size: 12, italics: true);
+                Text = $@"{country.FullName}";
```

Comment:
Not too much benefit from interpolating a single property. Unless you wanted the coercion to empty string logic, at which point

```suggestion
                Text = country.FullName ?? string.Empty;
```

is probably way easier to process. The interpolation would probably get optimised out anyway but it looks very strange.